index.js

```diff
@@ -25,6 +25,9 @@ FakeMap.prototype = {
   },
 };
 
+// Non-enumerable keys to include in comparison of Error objects
+var EXTRA_ERROR_KEYS = [ 'name', 'message' ];
```

Comment:
Perhaps we could move this to be just above `addExtraErrorKeys` since it's only used there?

Also, I wast thinking that these could be used inline on `addExtraErrorKeys` or even just declared inside that function since they are only used there if that doesn't hurt performance.